mat64/matrix_test.go

```diff
@@ -38,10 +38,19 @@ func panics(fn func()) (panicked bool, message string) {
 }
 
 func flatten(f [][]float64) (r, c int, d []float64) {
-	for _, r := range f {
-		d = append(d, r...)
+	r = len(f)
+	if r == 0 {
+		panic("bad test: no row")
```

Comment:
It's not an issue with flatten though. I want the message to be to either the author of the test that they wrote a bad test (or to an unfortunate client that got code through a mistake in review that it isn't their fault).